assets/queries/ansible/config/allow_unsafe_lookups_enabled/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+import data.generic.ansible as ansLib
+
+CxPolicy[result] {
+	defaultsGroup := input.document[i].groups.defaults
+
+    defaultsGroup.allow_unsafe_lookups == true
+
+	result := {
+		"documentId": input.document[i].id,
+		"resourceName": "defaults",
+		"resourceType": "n/a",
+		"searchKey": "defaults.allow_unsafe_lookups",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "allow_unsafe_lookups should be set to 'False'",
+		"keyActualValue": "allow_unsafe_lookups is set to 'True'",
```

Comment:
```suggestion
		"keyActualValue": "allow_unsafe_lookups is set to 'True'",
		"searchLine": common_lib.build_search_line(["groups", "defaults"], ["allow_unsafe_lookups"]),
```

Reply:
Hi! 
For Ansible Hosts and Config files, there is no need to fill this parameter since the attributes can vary in different files